templates/overrides/menu/menu--main.html.twig

```diff
@@ -4,6 +4,6 @@
  */
 #}
 {{ pattern('navigation', {
-  'variant': 'default',
+  'variant': 'navbar',
```

Comment:
We should set here class 'me-auto' and ditch the wrapping div in pattern-navbar.
If we do that and we override the block template for the main menu to remove the nav wrapper then we would have the needed structure in navbar